gradle/publish.gradle

```diff
@@ -70,10 +70,10 @@ publishGhPages.dependsOn prepareGhCredentials
 
 task prepareGradlePluginsRepoRelease {
     doLast {
-        // force evaluation of properties to check whether they're there or not
-        buildProperties.secrets['gradle.publish.key'].string
-        buildProperties.secrets['gradle.publish.secret'].string
-        System.properties['com.gradle.login.properties.file'] = rootProject.file('secrets.properties')
+        String key = buildProperties.env['GRADLE_PLUGINS_REPO_KEY'].or(buildProperties.secrets['gradle.publish.key']).string
+        String secret = buildProperties.env['GRADLE_PLUGINS_REPO_SECRET'].or(buildProperties.secrets['gradle.publish.secret']).string
+        project.setProperty('gradle.publish.key', key)
```

Comment:
Should that be `rootProject`? 🤔

Reply:
I think project because we apply the publish plugin to project not root. I may be wrong :)